providers/fab/docs/auth-manager/token.rst

```diff
@@ -136,16 +136,12 @@ This custom logic overrides the default ``create_token`` method from the FAB aut
                     "role_keys": groups,
                 }
 
-                user = self.security_manager.auth_user_oauth(userinfo)
+                user = self.security_manager.auth_user_oauth(userinfo, rotate_session_id=False)
 
             # Fall back to the default implementation
             else:
                 user = super().create_token(headers=headers, body=body)
 
             log.info("User: %s", user)
 
-            # Log user into the session
-            if user is not None:
```

Comment:
I think we need this for users who are logging in with oauth through the UI (not the API). We need to store the user in session. I guess we need to create another new parameter, same fashion as `rotate_session_id`, to not store the user in session in some cases